### ISSUE TITLE
Could you modify voxel generation more quickly?

### ISSUE THREAD

**Issue by chadorii** [jantepya/Unity-Sprite-Voxelizer#5]
In VoxelUtils.cs
foreach (var color in colorMap)
{
    for (int i = 0; i < size; i++)
    {
        textureMap.SetPixel(i, color.Value, color.Key);
     }
}

That code was slow When there are many colors
Could you modify voxel generation more quickly? Something Like using SetPixel32

Grammar may be strange because I am not familiar with English.

**Comment by jantepya** [comment]
Yup! Using SetPixels32 with a color array is more efficient than calling SetPixel for each individual pixel. I added some optimizations for that in #6, though I don't think it will have too much effect on overall performance.
I'll try to think of some other ways to improve performance.
